File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/DataViz/Line_Charts/Local_DataController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Line_ChartsController : Controller
    {
        public ActionResult Local_Data()
        {
            return View(ChartDataRepository.InternetUsers());
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/linecharts/LocalDataController.java
package com.kendoui.spring.controllers.linecharts;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-line_chart-local_data-controller")
@RequestMapping(value="/dataviz/line-charts/")
public class LocalDataController {
    @RequestMapping(value = "/local-data", method = RequestMethod.GET)
    public String index(Model model) {
        model.addAttribute("internetUsers", ChartDataRepository.InternetUsers());
     
        return "/dataviz/line-charts/local-data";
    }
}
